junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/ExtensionContext.java

```diff
@@ -274,6 +274,16 @@ default void publishReportEntry(String key, String value) {
 		this.publishReportEntry(Collections.singletonMap(key, value));
 	}
 
+	/**
+	 * Publish the specified message to be consumed by an
+	 * {@code org.junit.platform.engine.EngineExecutionListener}.
+	 *
```

Comment:
Document that this convenience overload uses `message` as the `key` for publishing a report entry here too.